### ISSUE TITLE
Feature request: support video links

### ISSUE THREAD

**Issue by darsnack** [notaapp/nota#172]
Currently, only linking to image files is supported. When suggesting Nota to someone on Reddit, they asked about inline (embedded) video for live preview. I tested this out, and it appears that only `[Text](video-file.mp4)` works as a text link to the file. Using `![Caption](video-file.mp4)` does not work for either preview or inline preview.

**Comment by astoilkov** [comment]
Yes, that's a good idea. We should also support previewing other types of media like PDF and audio.
